root/src/main/java/com/example/servingwebcontent/MovieListController.java

```diff
@@ -23,4 +29,28 @@ public String login() {
 	public String register() {
 		return "register";
 	}
+
+	@GetMapping("/test")
+	public String test() throws java.io.FileNotFoundException {
+		BufferedReader reader;
+		String user = "", pass = "";
+	
+		try {
+			reader = new BufferedReader (new FileReader("./userInfo.txt"));
+			String line = reader.readLine();
+			user = line;
+			line = reader.readLine();
+			pass = line;
+			reader.close();
+		} catch (IOException e) { System.out.println(e); }
+
+		ConnectDatabase connection = new ConnectDatabase("movielist", user, pass);
+
+		MovieObject newMovie = new MovieObject("4", "Movie", "Titanic", "1997-12-19", 195, "PG-13");
```

Comment:
You should have a try, finally block here so you always close the connection even if there is an error.